src/containers/SearchForHelpPage/index.jsx

```diff
@@ -1,24 +1,80 @@
-import React from "react";
+import React, { useState } from "react";
 import SeekHelpMap from "../../components/SeekHelpMap";
 import SeekHelpSingleCard from "../../components/SeekHelpSingleCard";
 import { Container, Row, Col, Dropdown, DropdownButton } from "react-bootstrap";
 import { useTranslation } from "react-i18next";
 import "./index.scss";
 
+const CITIES_WITH_HELP = [
+  {
+    name: "Istanbul",
+    coordinates: {
+      lat: 41.015137,
+      lng: 28.97953,
+    },
+  },
+  {
+    name: "Ankara",
+    coordinates: {
+      lat: 39.92077,
+      lng: 32.85411,
+    },
+  },
+  {
+    name: "Izmir",
+    coordinates: {
+      lat: 38.41885,
+      lng: 27.12872,
+    },
+  },
+];
+
 const SearchForHelpPage = () => {
   const { t } = useTranslation();
+  const [seekHelpCards, setSeekHelpCards] = useState(
+    t("home.seekHelpSection", { returnObjects: true })
+  );
+  const [filteredCity, setFilteredCity] = useState({
+    name: "Istanbul",
```

Comment:
there is some code duplication here that is dangerous. say, in case you change where the pin is in istanbul. you have to change the same constant in 2 places = one of the cardinal rules of writing our code violated

useState(CITIES_WITH_HELP[0])

Reply:
if i were writing this code, to be ultra defensive, i would actually do:

useState(CITIES_WITH_HELP.find(c => c.name === "Istanbul") || CITIES_WITH_HELP[0])

this says:
1. look for a city called istanbul, use it ("what if my teammate changes Istanbul to be the third city, or adds a city before it?")
2. what if there is no city called Istanbul ("what if my teammate lowercases it for some reason?"). in that case, prefer the first city on the list
3. if some idiot deletes the whole list then CITIES_WITH_HELP[0] will be undefined i suppose, at that point the code deserves to crash

Reply:
okay, makes a lot of sense. i will fix it. thank you for your feedback!